src/content/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management.mdx

```diff
@@ -18,19 +18,23 @@ Before using our SCIM API, you must first [enable SCIM for an authentication dom
 
 Note that after you enable SCIM, there are necessary [next steps](#next-steps) to do, including downgrading some users to basic users, and granting user groups access to New Relic accounts. 
 
+## Tutorial [#tutorial]
+
+If you want to see more specific instructions on how to use this API, see the [SCIM API tutorial](/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim).
```

Comment:
I'd rather front load this with the link. Something like:
"See the SCIM API tutorial for more specific information on using this API."